src/doc-printer.js

```diff
@@ -411,6 +416,16 @@ function printDocToString(doc, options) {
       }
     }
   }
+
+  const cursorPlaceholderIndex = out.indexOf(cursor.placeholder);
+  if (cursorPlaceholderIndex >= 0) {
```

Comment:
fixed in https://github.com/prettier/prettier/pull/1637/commits/c4e08d6eedb7a2500972fdf711e936f165b8dcd6